packages/components/src/components/FileUploader/FileUploader.tsx

```diff
@@ -46,6 +46,9 @@ export interface FileUploaderProps
 
   /** Disables the file uploader */
   disabled?: boolean;
+
+  /** When `true`, indicates that the user must upload a file before the form can be submitted. */
+  required?: boolean;
```

Comment:
Do we need the required prop here? Don't we actually need it in the FileUploaderButton so that it gets applied to the hidden input?

Reply:
There are some places that are not using the <FileUploaderButton /> together with FileUploader. Maybe we should have both options?